vulnerabilities/msgvalue-loop.md

```diff
@@ -0,0 +1,44 @@
+# Using ``msg.value`` in a loop.
```

Comment:
Can we remove the period from the title here? Maybe also capitalize "Loop" to be nicely formatted

Reply:
Fixed in the latest commit!